example.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
```

Comment:
I see. I'm not very keen of putting even more stuff at the root level of each repo.